test/src/sdk-harness/test_projects/registers/src/main.sw

```diff
@@ -3,47 +3,51 @@ contract;
 use std::context::{*, call_frames::*, registers::*};
 
 abi Registers {
+    /*
+    fn get_program_counter() -> raw_ptr;
+    fn get_stack_start_ptr() -> raw_ptr;
+    fn get_stack_ptr() -> raw_ptr;
+    fn get_frame_ptr() -> raw_ptr;
+    fn get_heap_ptr() -> raw_ptr;
+    fn get_instrs_start() -> raw_ptr;
+    */
     fn get_overflow() -> u64;
-    fn get_program_counter() -> u64;
-    fn get_stack_start_ptr() -> u64;
-    fn get_stack_ptr() -> u64;
-    fn get_frame_ptr() -> u64;
-    fn get_heap_ptr() -> u64;
     fn get_error() -> u64;
     fn get_global_gas() -> u64;
     fn get_context_gas() -> u64;
     fn get_balance() -> u64;
-    fn get_instrs_start() -> u64;
     fn get_return_value() -> u64;
     fn get_return_length() -> u64;
     fn get_flags() -> u64;
 }
 
 impl Registers for Contract {
-    fn get_overflow() -> u64 {
-        overflow()
-    }
-
-    fn get_program_counter() -> u64 {
+    /*
+    fn get_program_counter() -> raw_ptr {
```

Comment:
How come you've had to disable these tests?

Reply:
It's not currently possible to return raw pointers back the rust through the SDK. We need to discuss how this is supposed to work. For instance if someone returns a `Vec` then they probably want to allocate on the rust side and copy the data across rather than returning a (wrapped) `u64` representing the pointer. So for now the parts of the SDK tests that return raw pointers have been disabled.